cache_registry/sync/commands.py

```diff
@@ -154,6 +156,39 @@ def print_all_undertakings(undertakings):
     print(undertakings_count, "values")
 
 
+@sync_manager.command("auditors")
+@click.option("-d", "--days", "days", help="Number of days the update is done for.")
+@click.option("-u", "--updated", "updated_since", help="Date in DD/MM/YYYY format")
+@click.option("-p", "--page_size", "page_size", help="Page size")
+@click.option("-i", "--auditor_uid", "uid", help="Auditor uid")
+def auditors(days=7, updated_since=None, page_size=200, uid=""):
+    return call_auditors(days, updated_since, page_size, uid)
+
+
+def call_auditors(days=7, updated_since=None, page_size=200, uid=""):
+    if not uid:
+        last_update = get_last_update(days, updated_since, model_name=Auditor)
+    else:
+        last_update = None
+        print(f"Fetching data for auditor {uid}")
+
+    auditors_data = get_latest_auditors(
+        updated_since=last_update, page_size=page_size, uid=uid
+    )
+    for auditor_dict in auditors_data:
+        update_auditor(auditor_dict)
+
+    cleanup_unused_users()
+
+    if not uid:
+        auditors_count = len(auditors_data)
+        log_changes(last_update, auditors_count, domain=FGAS)
```

Comment:
This log is just for Undertaking updates. I don't think auditors need logs in the DB for now.